lib/rubocop/cop/mixin/space_after_punctuation.rb

```diff
@@ -19,9 +19,14 @@ def investigate(processed_source)
         end
       end
 
+      def space_style_before_rcurly?
```

Comment:
I can't find that we've used this technique before, checking "calling an abstract method". It's code that's impossible (more or less) to cover in tests. I don't think we need it. We'll find out in testing if any implementation is missing, even without it.